docker/message_broker/Dockerfile

```diff
@@ -1,9 +1,12 @@
 # Based from https://github.com/rmohr/docker-activemq
-FROM openjdk:8-jre
+FROM registry.access.redhat.com/ubi8/ubi:latest
 
 ENV ACTIVEMQ_VERSION 5.15.12
 ENV ACTIVEMQ_HOME /opt/activemq
 
+# keeping container image updated (security updates)
+RUN  dnf install java-1.8.0-openjdk maven -y && dnf update -y
```

Comment:
Hey Jan,
Do you know if we specifically need Java 1.8?, I believe we have the latest Java 11 and Java 17 available as well.
I am thinking that maybe using latest java will give less security issues

Reply:
That is good point! I will check that.